addons/im_livechat/static/tests/livechat_channel_info_list.test.js

```diff
@@ -65,24 +65,24 @@ test("editing livechat note is synced between tabs", async () => {
     await openDiscuss(channelId, { target: tab1 });
     await openDiscuss(channelId, { target: tab2 });
     await contains(
-        ".o-livechat-ChannelInfoList textarea",
-        { value: "Initial note" },
-        { target: tab1 }
+        `.o-mail-Discuss-asTabContainer[data-as-tab-id=${tab1.discussAsTabId}] .o-livechat-ChannelInfoList textarea`,
```

Comment:
This is a bit verbose, we might want to put `.o-mail-Discuss-asTabContainer[data-as-tab-id=${tab1.discussAsTabId}]` in a variable once, and just use `${tab1}` or `${tab2}` in the actual test.